raiden/channel.py

```diff
@@ -553,15 +553,18 @@ def __init__(
 
         self.received_transfers = []
         self.sent_transfers = []  #: transfers that were sent, required for settling
+        self.STATE_OPENED = 'opened'
```

Comment:
I would recommend having those as static class attributes and not as members of the class instance.

To understand what I mean read [here](http://stackoverflow.com/questions/68645/static-class-variables-in-python).

No need to initialize every new channel instance with these 3 constants. Just move them to the class body and as such they should be accessible as: `Channel.STATE_OPENED` e.t.c.